setup.cfg

```diff
@@ -1,6 +1,7 @@
 [versioneer]
 VCS = git
 style = pep440-post
-versionfile_source = suitcase/mongo/_version.py
-versionfile_build = suitcase/mongo/_version.py
+versionfile_source = suitcase/mongo_normalized/_version.py
+versionfile_build = suitcase/mongo_embedded/_version.py
```

Comment:
These two should be the same. We just have to pick one -- doesn't matter which. The "source of truth" is the git tag, which will be the same for both.

Reply:
updated